make -j1 fails: src/game/gamecore.h:12:10: fatal error: generated/protocol.h: No such file or directory

```
In file included from /ddnet-source/src/game/gamecore.cpp:3:
/ddnet-source/src/game/gamecore.h:12:10: fatal error: generated/protocol.h: No such file or directory
   12 | #include <generated/protocol.h>
      |          ^~~~~~~~~~~~~~~~~~~~~~
compilation terminated.
```
With more parallelism it works, so I assume the dependency between gamecore.cpp -> protocol.h isn't encoded properly in cmake.

This is causing the official (nightly) builds to fail.

--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -2203,6 +2203,9 @@ list(REMOVE_ITEM GAME_SHARED ${ENGINE_UUID_SHARED})
 set(GAME_GENERATED_SHARED
   src/generated/data_types.h
   src/generated/git_revision.cpp
+  src/generated/protocol.h
+  src/generated/protocol7.h
+  src/generated/protocolglue.h
 )
 set(DEPS ${DEP_JSON} ${DEP_MD5} ${ZLIB_DEP})
 

